backend/decompme/settings.py

```diff
@@ -165,7 +183,7 @@
 LOCAL_FILE_DIR = BASE_DIR / "local_files"
 
 USE_SANDBOX_JAIL = env("USE_SANDBOX_JAIL")
-if sys.platform == "darwin":
+if sys.platform == "darwin" or (sys.platform == "win32" and not in_wsl()):
```

Comment:
Is the `not in_wsl` check redundant here? Is the following more correct?

```py
# always disable sandbox on mac and windows
if sys.platform == "darwin" or sys.platform == "win32"):
```

Reply:
I wasn't sure if wsl still technically reports as being windows in some system properties. If this suffices, then yeah, that's better. I'll change it